azurerm/internal/services/web/app_service.go

```diff
@@ -809,7 +809,7 @@ func schemaAppServiceIpRestriction() *schema.Schema {
 				"ip_address": {
 					Type:         schema.TypeString,
 					Optional:     true,
-					ValidateFunc: validate.CIDR,
+					ValidateFunc: validate.CIDRv4n6,
```

Comment:
I think the `validation.IsCIDR` function should work here? its:
```
// IsCIDR is a SchemaValidateFunc which tests if the provided value is of type string and a valid CIDR
func IsCIDR(i interface{}, k string) (warnings []string, errors []error) {
	v, ok := i.(string)
	if !ok {
		errors = append(errors, fmt.Errorf("expected type of %s to be string", k))
		return warnings, errors
	}

	if _, _, err := net.ParseCIDR(v); err != nil {
		errors = append(errors, fmt.Errorf("expected %q to be a valid IPv4 Value, got %v: %v", k, i, err))
	}

	return warnings, errors
}
```

Reply:
As validation package does not have IsCIDR I've added it there and update the schema validation.